test/check_mongo_orchestration_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Should we get rid of this file altogether now that there's a `js` version?

Reply:
I'll remove it

Reply:
I think this file still needs to be removed